mfr/extensions/tabular/exceptions.py

```diff
@@ -33,6 +33,14 @@ def __init__(self, message, *args, code: int=400, **kwargs):
         super().__init__(message, *args, code=code, **kwargs)
         self.attr_stack.append([self.__TYPE, {}])
 
+class TooBigToRenderError(TabularRendererError):
+
+    __TYPE = 'tabular_too_big_to_render'
+
+    def __init__(self, message, *args, code: int=400, **kwargs):
+        super().__init__(message, *args, code=code, **kwargs)
+        self.attr_stack.append([self.__TYPE, {}])
```

Comment:
One bit of data that would be useful to track for the metrics would be the received size and the settings limit, both as `int`s.  Down the road, we can check our analytics and see how often people are banging into these limits and if they should be adjusted.